### DISCUSSION TITLE
Mandatory headings in theorems

### DISCUSSION THREAD

**Discussion by bwelman** [quarto-dev/quarto-cli#7289] [Q&A] [answered]
### Description

Since v1.4.404, the handling of theorems in a book project has changed. It is now mandatory to have a first heading in the block. Without a header you get the error
.../main.lua:9025: attempt to index a nil value (field 'integer index')

Until v1.4.398 you could omit this header. Could this possibly have something to do with issue #7148?

I have a textbook with many exercises, all without a header in an #exr- block. Do I have to provide all of these with a meaningless and unwanted header?

**Comment by mcanouil** [comment, marked as answer]
See:
- #7259

**Reply by cderv** [reply]
Thanks ! 

It has been solved now and should be working in latest 1.4 version
